rostwitter/scripts/tweet.py

```diff
@@ -28,7 +28,7 @@ def __init__(self):
 
     def tweet_cb(self, msg):
         message = msg.data
-        rospy.loginfo(rospy.get_name() + " sending %s", message)
+        rospy.logdebug(rospy.get_name() + " sending %s", message)
```

Comment:
if  `loginfo` is not noisy  without bsae64 message, how abotut use loginfo + something like `' '.join([a if len(a) < 32 else a[0:32]+'...'  for a in txt.split(' ')])` ?

If we already feel darty without base64, using `logdebug` make sense